pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -302,7 +303,11 @@ def _save_model(self, filepath: str, trainer, pl_module):
 
         # delegate the saving to the trainer
         if self.save_function is not None:
-            self.save_function(filepath, self.save_weights_only)
+            # Todo (tudorcebere): Add support for saving the model   # noqa E265
+            if _PYSYFT_AVAILABLE:
+                pass
+            else:
+                self.save_function(filepath, self.save_weights_only)
```

Comment:
This means no models will be saved if syft is installed at all, regardless of it it's being used

Reply:
Yes, it isn't supported yet.

Reply:
But I agree with @carmocca that this shouldn't be the right type of check here. We should somehow check if it's used not if it's just installed

Reply:
You are entirely right ! Let me work on it.